modules/cas_disk/exp_obj.c

```diff
@@ -530,7 +529,12 @@ int casdsk_exp_obj_create(struct casdsk_disk *dsk, const char *dev_name,
 	exp_obj->owner = owner;
 	exp_obj->ops = ops;
 
-	gd = alloc_disk(1);
+	result = _casdsk_init_tag_set(dsk, &dsk->tag_set);
+	if (result) {
+		goto error_init_tag_set;
+	}
+
+	gd = blk_mq_alloc_disk(&dsk->tag_set, NULL);
```

Comment:
This needs additional build config, otherwise it will break building on older kernels.